app/controllers/clients_controller.rb

```diff
@@ -21,8 +22,8 @@ def edit; end
 
   def create
     @client = Client.new(client_params)
-    @client.user = current_user
     authorize @client
+    @client.user = current_user
```

Comment:
All assignments should be done before authorization, in case the policy wants to look at `#user`.